language-server/test/dotty/tools/languageserver/CompletionTest.scala

```diff
@@ -308,40 +308,140 @@ class CompletionTest {
 
   @Test def completeExtensionMethodWithTypeParameter: Unit = {
     code"""object Foo
-          |extension [A](foo: Foo.type) def xxxx: Int = 1
+          |extension (foo: Foo.type) def xxxx[A]: Int = 1
           |object Main { Foo.xx${m1} }""".withSource
       .completion(m1, Set(("xxxx", Method, "[A] => Int")))
```

Comment:
This is intentional. This is actually a new test case. The old one got copied below and renamed to `completeExtensionMethodFromExtensionWithTypeParameter` because now we need to distinguish between the two